libs/screen-orientation/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "@ng-web-apis/screen-orientation",
+    "version": "3.0.0",
+    "description": "A library for declarative use of screen orientation with Angular",
+    "keywords": [
+        "angular",
+        "ng",
+        "screen",
+        "orientation"
+    ],
+    "homepage": "https://github.com/taiga-family/ng-web-apis/blob/main/libs/intersection-observer/README.md",
+    "bugs": "https://github.com/taiga-family/ng-web-apis/issues",
+    "repository": "https://github.com/taiga-family/ng-web-apis",
+    "license": "Apache-2.0",
+    "author": {
+        "name": "Alexander Inkin",
+        "email": "alexander@inkin.ru"
+    },
+    "contributors": [
+        "Roman Sedov <79601794011@ya.ru>"
+    ],
+    "peerDependencies": {
+        "@angular/core": ">=12.0.0",
+        "@ng-web-apis/common": ">=2.0.0"
```

Comment:
Shouldn't it be 3? Should we also include rxjs?

Reply:
fixed